BruteShark/PcapAnalyzer/Modules/PasswordsModule/CredentialsParsers/FtpPasswordParser.cs

```diff
@@ -6,8 +6,8 @@
 namespace PcapAnalyzer
 {
     public class FtpPasswordParser : IPasswordParser
-    {
-        private Regex _ftpSuccessfullLoginRegex = new Regex(@"220(.*)\r\nUSER\s(?<Username>.*)\r\n331(.*)\r\nPASS\s(?<Password>.*)\r\n");
+    {   
+        private Regex ftpSuccessfullLoginRegex = new Regex(@"220(.*)"+Environment.NewLine+ @"USER\s(?<Username>.*)"+Environment.NewLine+@"331(.*)" + Environment.NewLine + @"PASS\s(?<Password>.*)" + Environment.NewLine);
```

Comment:
There is a problem here: we use the user operating system new line format, but the data we get can be from any OS. I guess that was a potential bug before this PR.  I believe a Regex that catch both formats will be better

Reply:
you are right